data/data-pipeline/data_pipeline/etl/sources/national_risk_index/etl.py

```diff
@@ -99,28 +101,71 @@ def transform(self) -> None:
             inplace=True,
         )
 
+        # Only use disasters linked to climate change
+        disaster_categories = [
+            "AVLN",  # Avalanche
+            "CFLD",  # Coastal Flooding
+            "CWAV",  # Cold Wave
+            "DRGT",  # Drought
+            "HAIL",  # Hail
+            "HWAV",  # Heat Wave
+            "HRCN",  # Hurricane
+            "ISTM",  # Ice Storm
+            "LNDS",  # Landslide
+            "RFLD",  # Riverine Flooding
+            "SWND",  # Strong Wind
+            "TRND",  # Tornado
+            "WFIR",  # Wildfire
+            "WNTW",  # Winter Weather
+        ]
+
+        self.df = df_nri
```

Comment:
Hmm, this seems totally off to me. Why did you move this up here from line 122? You're essentially ensuring that none of your work gets saved?

Reply:
@lucasmbrown-usds Good catch - I forgot to move that back down!!